### ISSUE TITLE
reproduce

### ISSUE THREAD

**Issue by wangtao0219** [ultralytics/ultralytics#1801]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I modified the parameters according to the configuration of yolov8n.pt, except that batch can only be set to a maximum of 64. （yolo detect train data=coco.yaml model=yolov8n.ml epochs=500 batch=64 cache=disk conf=0.001 half=True opset=17 weight_decay=0.001 v5loader=True）
In this case, what is missing if I want to reproduce the 37.3% detection performance of yolov8n on the coco dataset?

**Comment by glenn-jocher** [comment]
@wangtao0219 setting the batch size to 64 is already quite large, so I would suggest looking into other ways to improve performance. You can try adjusting other hyperparameters, such as the learning rate or weight decay. You can also try using larger image sizes for training and testing, and considering data augmentation to improve the model's ability to detect objects. Additionally, make sure that you are using the latest version of the code and that your dependencies are up to date. You may also want to look into using a different pre-trained model or fine-tuning the one you are currently using.

**Comment by wangtao0219** [comment]
@glenn-jocher 
My question is reproduce YOLOv8 experimental results (train from scratch) on coco(train2017/val2017)
（yolo detect train data=coco.yaml model=yolov8n.ml epochs=500 batch=64 cache=disk conf =0.001 half=True opset=17 weight_decay=0.001 v5loader=True）I have the same configuration as the official training yolov5n.pt, but I don't get 37.3% result, I would like to know what other parameters should be added?

**Comment by wangtao0219** [comment]
@glenn-jocher       I don't get 37.3% result.
Should I add cos_lr=True weight_decay=0.0005？ 
Also my picture size is 640, should I set it bigger, what is it?
I'm using the default data augmentation, how do I change it?

**Comment by glenn-jocher** [comment]
@wangtao0219 It's possible that adjusting the learning rate schedule (such as using the cosine learning rate annealing method with cos_lr=True) and weight decay value (like setting it to 0.0005) could help improve the model's performance. Increasing the image size may also be helpful, although you should keep in mind that larger images will require more computing power and may result in longer training times. To change the data augmentation, you can modify the augmentation settings in the `data.yaml` file. It's important to experiment with different hyperparameters and settings to find the best combination for your specific dataset and use case.

**Comment by wangtao0219** [comment]
@glenn-jocher 
Please, is there a specific process of yolov8 model training that I can view? For example, the wandb of yolov5

**Comment by wangtao0219** [comment]
@glenn-jocher 
Is it necessary to wait for the specific training parameters to come out together with the paper, roughly when is it ?

**Comment by glenn-jocher** [comment]
@wangtao0219 Ultralytics hasn't released a specific training process for YOLOv8 yet, but you can refer to the code and configuration files provided in the repository for guidance. As for the release of the specific training parameters, it's difficult to give a definitive answer, as this can vary depending on various factors such as the complexity of the model and the availability of resources. However, you should keep an eye on the Ultralytics Github page and other official channels for updates, as they may release more information in the future. In the meantime, it's important to experiment with different hyperparameters and settings to find the best combination for your specific use case.
